core/src/main/resources/schema/app/20190522120735_CB-1513_remove_kerberos_env_relation.sql

```diff
@@ -0,0 +1,16 @@
+-- // CB-1513 remove kdc env relation
+-- Migration SQL that makes the change goes here.
+DROP TABLE IF EXISTS env_kdc;
```

Comment:
This also drops the indexes that refer to this table right?

Reply:
most likely

Reply:
are you suggest that to drop the indexes beforehand?

Reply:
only if it does not do it

Reply:
it does